src/platform-includes/crons/setup/python.celery.mdx

```diff
@@ -1,46 +1,92 @@
-## Job Monitoring
+## Celery Beat Auto Discovery
 
-Use the Celery Beat integration to monitor and notify you if your job is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
+Use the Celery integration to monitor your [Celery periodic tasks](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html) and get notified when a task is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
 
-First, set up your Celery beat configuration:
+First, set up your Celery beat schedule:
 
 ```python
-from celery import Celery, signals
+# tasks.py
+from celery import Celery
 from celery.schedules import crontab
 
-import sentry_sdk
-from sentry_sdk.crons import monitor
-from sentry_sdk.integrations.celery import CeleryIntegration
 
-app = Celery('mytasks', broker='redis://localhost:6379/0')
+app = Celery('tasks', broker='...')
 app.conf.beat_schedule = {
     'set-in-beat-schedule': {
         'task': 'tasks.tell_the_world',
         'schedule': crontab(hour='10', minute='15'),
-        'args': ("in beat_schedule set", ),
+        'args': ("Some important message!", ),
     },
 }
 ```
 
-Initialize Sentry either in `celeryd_init` or `beat_init` signal:
+Initialize Sentry in the `celeryd_init` or `beat_init` signal. Make sure to pass your Celery `app` variable to the `CeleryIntegration`:
 
 ```python
-#@signals.celeryd_init.connect
-@signals.beat_init.connect
+# tasks.py
+from celery import signals
+
+import sentry_sdk
+from sentry_sdk.integrations.celery import CeleryIntegration
+
+
+#@signals.beat_init.connect
+@signals.celeryd_init.connect
 def init_sentry(**kwargs):
     sentry_sdk.init(
         dsn='___PUBLIC_DSN___',
-        integrations=[CeleryIntegration()],
+        integrations=[CeleryIntegration(celery_app=app)],  # 👈 pass Celery app.
         environment="local.dev.grace",
-        release="v1.0.7-a1",
+        release="v1.0",
     )
 ```
 
-Finally, link your Celery task to a Sentry Cron Monitor:
+This will now auto discover all the tasks in your Celery Beat schedule and will capture telemetry data when the task is started, when it finishes or fails.
```

Comment:
```suggestion
Once Sentry Crons is set up, tasks in your Celery Beat schedule will be auto-discoverable, and telemetry data will be captured when a task is started, when it finishes, and when it fails. 

```

Reply:
I edited this suggestion to lowercase "beat". AFAICT the extension is called "Celery Beat" but it refers to its worker and beat services with lowercase.